packages/propeller/src/Queue.ts

```diff
@@ -88,7 +88,7 @@ export class AsyncQueue<T> {
     const value = this.queue.tryFind(predicate)
     if (value) return value
     return new Promise<T>((resolve, reject) => {
-      const abort = () => reject(new Error("Aborted"))
+      const abort = () => reject(new DOMException("The operation was aborted", "AbortError"))
```

Comment:
@nordfjord why `DOMException` in this very backend-y package? It means it doesn't work with Node < 20, either.

Reply:
<img width="1256" alt="image" src="https://github.com/equinox-project/equinox-js/assets/804625/86edb381-858a-4522-8150-f4d217f90b96">

Reply:
Because that's what node throws from `AbortController#abort`

Reply:
Would be even worse to have multiple types of exceptions representing abort errors IMO

Reply:
See https://github.com/nodejs/node/blob/main/lib/internal/abort_controller.js#L178-L179